tests/assets/ansible/roles/node_base/tasks/sles-15.yml

```diff
@@ -28,11 +28,22 @@
         - name: "sle_server_updates"
           url: "http://download.suse.de/ibs/SUSE/Updates/SLE-Product-SLES/{{ sle_major }}-SP{{ sle_minor }}/x86_64/update/"
       loop_control:
-        loop_var: repo
+        loop_var: _repo
   vars:
     sle_major: "{{ ansible_distribution_version.split('.')[0] }}"
     sle_minor: "{{ ansible_distribution_version.split('.')[1] }}"
 
+- name: add extra repositories
+  zypper_repository:
+    name: '{{ _repo.key }}'
+    repo: '{{ _repo.value }}'
+    priority: 50
+    state: present
+    auto_import_keys: yes
+  loop: "{{ lookup('dict', repo, wantlist=True) }}"
```

Comment:
I think we should call this variable `extra_repos` or something

Reply:
Done